src/platform/testing/e2e/cypress/support/commands/expandAccordions.js

```diff
@@ -6,23 +6,14 @@ Cypress.Commands.add('expandAccordions', () => {
   Cypress.log();
 
   cy.get('main').then($main => {
-    // Check if va-accordion-item Web Component exists
-    if ($main.find('va-accordion-item').length > 0) {
-      cy.get('va-accordion-item')
+    const accordionMultiOpenSelector =
+      'va-accordion:not([open-single=""]):not([open-single])';
+    // Click "Expand all" button on va-accordion if it exists
+    if ($main.find(accordionMultiOpenSelector).length > 0)
+      cy.get(accordionMultiOpenSelector)
         .shadow()
-        .then(accordion => {
-          // If it exists and Accordions are not already expanded
-          if (accordion.find('button[aria-expanded=false]').length > 0) {
-            cy.get('va-accordion-item')
-              .shadow()
-              .find('button[aria-expanded=false]')
-              .each(button => {
-                // Click to open Accordions
-                cy.wrap(button).click({ force: true });
-              });
-          }
-        });
-    }
+        .find('button[aria-label="Expand all accordions"]')
```

Comment:
This implementation would throw an error if `expandAccordions` was called after the accordion was already expanded:

![image](https://user-images.githubusercontent.com/2008881/159049268-16726027-4ea4-431a-89c4-1cfe470a97ad.png)

Maybe this can be expected behavior.

Reply:
@bkjohnson yeah so the aria-label changes from `Expand all accordions` to `Collapse all accordions` on click which is why that error is occurring. We could change the aria-label to `Expand or Collapse all accordions` in the Web Component and that would solve the issue if you'd like